src/coreclr/vm/corelib.h

```diff
@@ -68,6 +68,19 @@
 #define DEFINE_FIELD_U(stringName, unmanagedContainingType, unmanagedOffset)
 #endif
 
+//
+// BEGIN_ILLINK_FEATURE_SWITCH and END_ILLINK_FEATURE_SWITCH allow IL linker to guard types behind a feature switch
+// Current support is only around class scope and not for standalone members of classes
```

Comment:
Can you add more details here? Maybe an example of the data that will be generated and/or links to docs. For someone not working regularly with the IL linker and feature switches, I don't think this comment makes it clear exactly how using these macros relates to types being rooted (or not) or where to look for that information.